apps/total-typescript/src/pages/tutorials/[module]/[exercise]/index.tsx

```diff
@@ -12,7 +12,12 @@ export const getStaticProps: GetStaticProps = async (context) => {
   const exercise = await getExercise(exerciseSlug)
 
   return {
-    props: {exercise, module},
+    props: {
+      exercise,
+      module,
+      videoThumbId: exercise.muxPlaybackId,
```

Comment:
exercises still exist and we can load them at build time and utilize the full set before treating it as a LessonResource at the page/component level